packages/cardano-services/src/cli.ts

```diff
@@ -361,6 +361,18 @@ withCommonOptions(
       .default(HANDLE_PROVIDER_SERVER_URL_DEFAULT)
       .argParser((serverUrl: string) => serverUrl)
   )
+  .addOption(
+    new Option('--use-typeorm-nft-metadata <true/false>', ProviderServerOptionDescriptions.useTypeormNftMetadata)
```

Comment:
Probably a better naming for this parameter is `--use-typeorm-assets`, but let me confront with @mkazlauskas before working on this

Reply:
`--use-typeorm-asset-provider`, following the existing pattern as we already have `--use-typeorm-stake-pool-provider`